.github/workflows/build_and_test.yml

```diff
@@ -67,13 +67,13 @@ jobs:
         run: >
           docker build -t integritee-cli-client-${{ matrix.flavor_id }}-${{ github.sha }}
           --target deployed-client
-          --build-arg WORKER_MODE_ARG=${{ matrix.mode }} --build-arg ADDITIONAL_FEATURES_ARG=${{ matrix.additional_features }}
+          --build-arg WORKER_MODE_ARG=${{ matrix.mode }} --build-arg ADDITIONAL_FEATURES_ARG=${{ matrix.additional_features }} --build-arg SGX_MODE=HW
```

Comment:
The client does actually not need any sgx mode as it does not run in a TEE.

Reply:
removed from cli build